<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Equilibreuse</class>
 <widget class="QMainWindow" name="Equilibreuse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Equilibreuse</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>281</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Consigne vitesse (en %)</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>235</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QDial" name="dial_ConsigneVItesse">
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_consigne">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="digitCount">
           <number>3</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_LancerMoteur">
          <property name="text">
           <string>Lancer Moteur</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Arreter">
          <property name="text">
           <string>ArrÃªter Moteur</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_vitesse">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>233</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Vitesse du moteur en tours par minutes</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetCourbes" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>731</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>480</y>
      <width>731</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_LancerAcquisition">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Lancer Acquisition</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Quitter">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1102</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Quitter</sender>
   <signal>clicked()</signal>
   <receiver>Equilibreuse</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>956</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_ConsigneVItesse</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber_consigne</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
